src/GitHub.App/Models/IssueCommentModel.cs

```diff
@@ -4,8 +4,9 @@ namespace GitHub.Models
 {
     public class IssueCommentModel : ICommentModel
     {
-        public string Body { get; set; }
         public int Id { get; set; }
+        public string NodeId { get; set; }
```

Comment:
As you noticed later, `NodeId` is the GraphQL ID which is different from the "database" Id. I didn't add doc comments to these fields because this will all change as we move move stuff to GraphQL, but I probably should have.